cmd/helm/history.go

```diff
@@ -77,6 +90,7 @@ func newHistoryCmd(c helm.Interface, w io.Writer) *cobra.Command {
 	f := cmd.Flags()
 	f.Int32Var(&his.max, "max", 256, "maximum number of revision to include in history")
 	f.UintVar(&his.colWidth, "col-width", 60, "specifies the max column width of output")
+	f.StringVarP(&his.outputFormat, "output", "o", "table", "prints the output in the specified format ('json' or 'yaml' or defaults to 'table')")
```

Comment:
can we change the docstring to "prints the output in the specified format (json|table|yaml)"? cobra will automatically display the default value, so it's kinda redundant:

```
><> ./bin/helm help history | grep -- --output
  -o, --output string        prints the output in the specified format ('json' or 'yaml' or defaults to 'table') (default "table")
```